libnavigation-ui/src/test/java/com/mapbox/navigation/ui/TestRouteProgressBuilder.java

```diff
@@ -32,18 +36,31 @@ RouteProgress buildTestRouteProgress(DirectionsRoute route,
       upcomingStepPoints = buildStepPointsFromGeometry(upcomingStepGeometry);
     }
 
+    // SBNOTE: this is probably not 100% correct but it's correct enough for the tests i'm working
```

Comment:
How are we stubbing in `libandroid-navigation-ui`? 👀 https://github.com/mapbox/mapbox-navigation-android/blob/master/libandroid-navigation-ui/src/test/java/com/mapbox/services/android/navigation/ui/v5/TestRouteProgressBuilder.java Doing the same but using the new [`RouteProgress`](https://github.com/mapbox/mapbox-navigation-android/blob/master/libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/RouteProgress.kt) model should be enough.

Reply:
I used that as a model but there isn't a one to one translation. I had to guess on a couple things but the unit tests pass without any modification so I think I got the right values. I put the comment there so that others wouldn't take it for granted that it was 100% correct (yet).